docs/manage-cloud/cloud-data-management.md

```diff
@@ -0,0 +1,80 @@
+---
+description: How manage your data on Cloud.
+contentType: howto
+---
+
+# Cloud data management
+
+Depending on your workload and on how you use n8n, your instance might risk reaching its memory capacity.
+
+If this happens, the instance faces constant crashes that render it virtually inaccessible.
+
+To avoid this, n8n recommends that you don't save unnecessary data, and build your workflows with memory efficiency in mind.
+
+## Data limits on each Cloud plan
+
+1. Current plans
+    * Trial - 320MiB RAM, 10 millicore CPU burstable;
+    * Starter - 320MiB RAM, 10 millicore CPU burstable;
+    * Pro-1 (10k executions) - 640MiB RAM, 20 millicore CPU burstable;
+    * Pro-2 (50k executions) - 1280MiB RAM, 80 millicore CPU burstable;
+    * Enterprise - 4096MiB RAM, 80 millicore CPU burstable.
+2. Legacy plans
+    * Start - 320MiB RAM, 10 millicore CPU burstable;
+    * Power - 1280MiB RAM, 80 millicore CPU burstable.
+
+## Data storage limits
+
+Each instance has up to 100GB of data storage allowance.
+
+## Execution data and binary data
+
+* **Execution data** is essentially all the structured data that pertains to the workflow's execution - Node data, parameters, variables, execution context and binary data references. It is text-based.
+
+* **Binary data** is non-textual data that cannot be represented as plain text. Essentially files and media - such as images, documents, audio files, videos, etc. It tends to be much larger than textual data.
+
+## Reduce memory consumption in your workflow
+
+The way workflows are built can influence how much data they will consume when executed. Although these guidelines are not applicable to all cases, they provide a baseline of best practices to avoid overconsumption of instance memory.
```

Comment:
```suggestion
The way you build your workflows affects how much data they consume when executed. Although these guidelines are not applicable to all cases, they provide a baseline of best practices to avoid over-consumption of instance memory.
```